<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite name="Suite" parallel="tests" thread-count="3">

	<!--<listeners>	
		<listener class-name="com.retry.Retryannotation"></listener>
	</listeners>-->
	
  <test  name="Test1">
	 <!-- <parameter name="browsername" value="edge"></parameter>-->
    <classes>
    <class name="testsuites.TestCase_CreateIndividualsErrorValidation"></class>
    </classes>
  </test> 
  
   <test  name="Test2">
	 <!-- <parameter name="browsername" value="edge"></parameter>-->
    <classes>
    <class name="testsuites.Testcase_OpportunityErrorValidationChain"></class>
    </classes>
  </test>
  
   <test name="Test3">
	 <!-- <parameter name="browsername" value="edge"></parameter>-->
    <classes>
    <class name="testsuites.Testcase_Worktype"></class>
    </classes>
  </test>
  
  
  
  
  
</suite> <!-- Suite -->
